airbyte-integrations/connectors/source-northpass-lms/source_northpass_lms/manifest.yaml

```
@@ -0,0 +1,768 @@
+version: "0.29.0"
```

```suggestion
version: "4.3.0"
```

Wow, this was old. Started with a generator? I need to clean this up today huh.